src/components/10-atoms/checkbox/ui.test.js

```diff
@@ -87,7 +87,7 @@ test('should style checked error checkbox-icon inner box correctly', async t =>
   await t
     .wait(50)
     .expect(await getIconBackgroundColor())
-    .eql('rgb(201, 20, 50)');
+    .eql('rgb(0, 0, 143)'); // has to be blue when seleted, due to styleguide specification
```

Comment:
The styleguide wants that? If true, my comments with links and screenshots is obsolete.

Reply:
Yes, this is what I found out while chatting with Thomas and looking at https://app.zeplin.io/project/5bed40045544f67ec5c16d69/screen/5ca617fd47f2319f5d7a6155.

Reply:
I also asked ken to approve from the designer side - and he did.